pulse2percept/electrode2currentmap.py

```diff
@@ -243,62 +251,74 @@ class Psycho2Pulsetrain(TimeSeries):
     Is used to generate pulse trains to simulate psychophysical experiments.
 
     """
-    def __init__(self, freq=20, dur=0.5, pulse_dur=.075/1000.,
-                 interphase_dur=.075/1000., delay=0., tsample=.005/1000.,
+    def __init__(self, freq=20, dur=0.5, pulse_dur=7.5e-5,
```

Comment:
I actually find `0.075/1000.` easier to parse. I know immediately that this is in seconds and that we've divided by 1000 because it's 0.075 milliseconds.